src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/WhitelistApiTest.java

```diff
@@ -44,7 +44,10 @@
      *
      * @throws Exception
      */
-    private void testGetAndPut(final int expectedStatus, final Header... headers) throws Exception {
+    private void testGetAndPut(final int expectedStatus, final boolean sendAdminCertificate, final Header... headers) throws Exception {
+
+        final boolean prevSendAdminCertificate = rh.sendAdminCertificate;
```

Comment:
Don't think you need to reset the value of  rh.sendAdminCertificate every time. Looks like this is being set everytime you call this method.